File: output/CodeGen/avatar/Java/Python/codeforces_519_B.py
import java.util.* ; import java.io.* ; 
def main ( ):
    sc = java.util.Scanner ( System.in )
    n = sc.nextInt ( )
    a = 0
    b = 0
    c = 0
    for i in range ( 0, n ):
        a += sc.nextInt ( )
    for i in range ( 0, n - 1 ):
        b += sc.nextInt ( )
    for i in range ( 0, n - 2 ):
        c += sc.nextInt ( )
    x = a - b
    y = b - c
    print x
    print y

# Run the program
main ( )

File: output/CodeLlama/avatar/Python/Java/codeforces_519_B.java
import java.util.Scanner;
public class codeforces_519_B {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] a = new int[n];
        int[] b = new int[n];
        int[] c = new int[n];
        int x = 0;
        int y = 0;
        int z = 0;
        for (int i = 0; i < n; i++) {
            a[i] = sc.nextInt();
        }
        for (int i = 0; i < n; i++) {
            b[i] = sc.nextInt();
        }
        for (int i = 0; i < n; i++) {
            c[i] = sc.nextInt();
        }
        for (int i = 0; i < n; i++) {
            x += a[i];
        }
        for (int i = 0; i < n; i++) {
            y += b[i];
        }
        for (int i = 0; i < n; i++) {
            z += c[i];
        }
        System.out.println(Math.abs(x - y));
        System.out.println(Math.abs(y - z));
    }
}